cli/command/registry.go

```diff
@@ -19,6 +20,10 @@ import (
 	"github.com/pkg/errors"
 )
 
+const patSuggest = "You can log in with your password or a Personal Access " +
+	"Token (PAT). Using a limited-scope PAT grants better security and is required " +
+	"for organizations using SSO. Learn more at https://docs.docker.com/go/access-tokens/"
```

Comment:
Perhaps use a string-literal for this (back-tick)

Reply:
Didn’t use them because we don’t want newlines